### PR TITLE
Callaaboutpage

### PR DESCRIPTION
**Pull request by callapatel**
about me page completed -- please check :pineapple:

### CODE REVIEW

**Comment by DeirdreAllison** [comment]
Hi @callapatel - I noticed you're tracking your .DS_Store file in git. It doesn't hurt anything, but it also doesn't do anything so if you'd like to prevent that file from being tracked in the future, here's some instructions: http://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository

I usually add it to my global ~/.gitignore just to keep my repos clean and avoid merge conflicts :)

File: app/views/welcome/about.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
...
           with the world premiere of the CODE documentary, and will continue to grow and
           expand in resources and functionality over time. Want to learn more or be a
           part of the she’s coding team? Contact us!
        </p>
        <p>
          Convert understanding into action – Join the She’s Coding team!
        </p>
    </div>
  </div>
</div>
```
Comment (natsteinmetz) [inline app/views/welcome/about.html.erb]: Below, you have a link to 'Home', next to the 'About' in the 'about-nav'. This should not be a link to home, but a link to 'How'. So under the same route you will get to 'About' and 'How'. 'How' will have content like the links to the tools we use for collaborating on the project (Github, Trello, Google Group, etc.), for now you can just add any dummy content.

So the idea is that one sees the 'About' section when clicking on 'About' in the main navbar. Then one sees the 'How' content when clicking on the 'How' link in the 'about-nav', and 'About' in that 'about-nav' turns into a link.

**Comment by natsteinmetz** [comment]
Hi @callapatel - great work! :)
I left you one comment inline - let me know if you have any questions regarding that.
And two more comments:
- Just as @DeirdreAllison said, checking in the .DS_Store file is not great, can you remove it from this PR?
- I don't think the font-sizes are set yet? (33px for the title, 26px for the body and 11px for the about/how links)?

**Comment by callapatel** [comment]
updated -- tell meif i missed anything! also removed to ds store.

**Comment by natsteinmetz** [comment]
@callapatel It looks like your schema.db file is off. Can you run db:migrate? Did you already drop and re-create your db, as I wrote in an earlier pull request (tagged you)? If not, what you should do is:
* Kill your rails server, and then in the console:
* dropdb shescoding_development
* createdb shescoding_development
* psql -d shescoding_development -c "GRANT ALL PRIVILEGES ON DATABASE shescoding_development to shescoding"
* rake db:migrate

That will fix your schema.db file.

**Comment by callapatel** [comment]
I did do that once before but I re ran it!

**Comment by natsteinmetz** [comment]
Awesome, thanks, the file looks good now!

**Pull request merged**